test/micro/org/openjdk/bench/java/util/stream/ops/ref/FindAny.java

```diff
@@ -60,4 +60,10 @@ public Long par_invoke() {
         return LongStream.range(0, size).parallel().boxed().findAny().get();
     }
 
+    public static void main(String... args) {
```

Comment:
Is this driver necessary?

Reply:
@cl4es Just wonder what your use case is for this addition. If this is accidentally committed, please remove it and I am glad with all other changes.

Reply:
Adding a `main` method in micros like these allow me to easily multi-purpose them as relatively clean, diagnostic startup microbenchmarks. It's something I've started adding to all micros I author. I guess I should do a write-up about it some time.

Reply:
Thanks for the explanation.